packages/flutter/test/rendering/annotated_region_test.dart

```diff
@@ -136,4 +136,137 @@ void main() {
       expect(parent.find<int>(const Offset(0.0, 0.0)), 1);
     });
   });
+  group('$AnnotatedRegion findAll', () {
```

Comment:
Even though it is covered by the listener tests, I think that we should add an test here which finds multiple values

Reply:
Done.